modules/data_integrity_flag/templates/menu_data_integrity_flag.tpl

```diff
@@ -0,0 +1,4 @@
+{*
+  Everything is rendered in index.js now!
+  I'm here because currently Loris breaks without me :(
+*}
```

Comment:
@jstirling91 I discovered that overriding `display()` in a modules that use new module structure actually breaks. @driusan wants to introduce a per-module flag that decides if smarty should be used, but we have to implement it before I can remove it